rdg_datasets/five_node_path_graph/import.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
I do not think that we need a special file for importing a single graph.  Is this the way we deal with other datasets?  It looks like too many lines for just providing a basic info about the graph.

Reply:
Yes, apparently. I followed the instruction.
`copy one of [migrate.py, import.py, generate.py] from another rdg to your <rdg-name> directory
modify the scripts variables to match your rdg`

Reply:
ok, seems a bit more complicated than needed, but fair.